### PR TITLE
Added related article cards between article body and comment section.

### PR DESCRIPTION
**Pull request by olofholm**
Added in the skeleton code for two cards in he article between the article body and the comment section. 
Just UI, nothing more.

### CODE REVIEW

**Comment by olofholm** [comment]
Switched the two placeholder cards for a horizontal scroll of multiple placeholder cards.

**Comment by ahunter135** [comment]
Not going to merge this one in yet, lets keep evolving it. Lets get related articles in the DB and then consume that data here and show them.

File: src/app/pages/news-article/news-article.page.html [hunk 1]
```diff
@@ -43,6 +43,27 @@
       <div [innerHTML]="article.content" ></div>
       <!--{{article.textBody}}-->
     </ion-card-content>
+
+    <ng-container *ngIf="checkIfAnyRelatedArticles()">
+      <h2 style="padding-left: 55px; padding-bottom: 5px;">Related Articles</h2>
```
Comment (ahunter135) [inline src/app/pages/news-article/news-article.page.html:48]: Can you move all styling to the .scss file? 

I'm sure I wasn't following that structure when this was being built, but it is actually best to not do inline styling and use classes/ids instead

**Comment by ahunter135** [comment]
We also need to do the same thing for trending

File: src/app/pages/news-article/news-article.page.html (hunk 1 above)
Comment (olofholm) [reply src/app/pages/news-article/news-article.page.html:48]: Yes, of course

**Comment by ahunter135** [comment]
Also need to be able to click on article to view it

**Pull request merged**